stdlib/2.7/os/__init__.pyi

```diff
@@ -197,8 +197,22 @@ def kill(pid: int, sig: int) -> None: ...
 def killpg(pgid: int, sig: int) -> None: ...
 def nice(increment: int) -> int: ...
 
-# TODO: plock, popen*, spawn*, P_*
-
+# TODO: plock, popen*, P_*
+
+def spawnl(mode: int, path: AnyStr, arg0: AnyStr, *args: AnyStr) -> int: ...
```

Comment:
Hm, technically each arg can independently be bytes or unicode, and ditto on PY3. But until someone complains I think this is fine. :-)